action/protocol/staking/handlers.go

```diff
@@ -141,9 +141,12 @@ func (p *Protocol) handleUnstake(ctx context.Context, act *action.Unstake, csm C
 		return log, fetchErr
 	}
 
-	bucket, fetchErr := p.fetchBucketAndValidate(csm, actionCtx.Caller, act.BucketIndex(), true, true)
-	if fetchErr != nil {
-		return log, fetchErr
+	bucket, rErr := p.fetchBucket(csm, act.BucketIndex())
+	if rErr != nil {
+		return log, rErr
+	}
+	if rErr = validateBucketOwner(bucket, actionCtx.Caller); rErr != nil {
+		return log, rErr
```

Comment:
1. the change should apply to `fetchBucketAndValidate()` with last input = false?
2. i think better modify inside `fetchBucketAndValidate`, instead of breaking up into 2 parts outside?